tests/test_backends.py

```diff
@@ -1,3 +1,7 @@
+# standard
```

Comment:
Should one of two "standard" be removed?

Reply:
Yes, fixed!